app/src/main/java/SmartModerationDesktopApp/ModerationCards/ModerationCardsController.java

```diff
@@ -66,15 +63,15 @@ public void receiveDeleteModerationCard(long cardId) {
             if (moderationCard.getCardId() == cardId) {
                 System.out.println("Found card to delete.");
                 moderationCards.remove(moderationCard);
-                mainWindow.getContentPane().remove(moderationCard);
+                MainWindow.getInstance().getContentPane().remove(moderationCard);
                 break;
             }
         }
     }
 
     public void receiveUpdateModerationCard(String message) {
```

Comment:
you can leave out the prefix "receive" in the moderationcardscontroller, as long its not an event handling method